ricecooker/sample_program.py

```diff
@@ -93,18 +103,18 @@ def guess_file_type(filepath, kind):
         "license": licenses.CC_BY_NC_SA,
         "files": [
             {
-                "path": "C:/Users/Jordan/Videos/testfolder/high resolution.mp4",
+                "path": "file:///C:/Users/Jordan/Videos/testfolder/high resolution.mp4",
                 # "ffmpeg_settings": {"max_width": 480, "crf": 20},
             },
-            {
-                "path": "C:/users/jordan/Videos/testfolder/captions.vtt",
-                "language": languages.getlang('en').code,
-            }
-            ,
-            {
-                "path": "file:///C:/users/jordan/Videos/testfolder/captions.vtt",
-                "language": languages.getlang('es').code,
-            }
+            # {
+            #     "path": "C:/users/jordan/Videos/testfolder/captions.vtt",
```

Comment:
Temporarily commented out since subtitles aren't allowed on the production server yet